src/libs/Browser/index.web.js

```diff
@@ -27,14 +27,21 @@ function getBrowser() {
     }
 
     match = match[1] ? match[1] : navigator.appName;
-    return match;
+    return match ? match.toLowerCase() : CONST.BROWSER.OTHER;
```

Comment:
Why do we need this change?

Reply:
because we were exporting it this way:
```js
export default () => {
    const browser = getBrowser();
    return browser ? browser.toLowerCase() : CONST.BROWSER.OTHER;
```
Now we are going to export more than one function, and it will keep the same behavior:
```js
export {
    getBrowser,
    isMobile,
};
```

Reply:
Gotcha